src/Components/Home/ListItem.js

```diff
@@ -10,16 +10,15 @@ function ListItem(prop) {
              <Row>
                 <Col>
                     {daysLeft == 0 ?  
-                        <Button variant="outline-dark" className="px-3"><i className="fas fa-shopping-cart"></i> Buy</Button>: 
+                        <Button variant="danger" className="px-3"><i className="fas fa-minus-circle"></i> Forgot</Button>: 
```

Comment:
Forget? in imperative clause.

Reply:
Forget over forgot but should it be forego instead?

Reply:
Forego describes the action better but it is  a tad advanced. If we want to target wider audience, including users whose primary language is not English, while not providing localization, we'd better stick with simpler vocabulary, imho.